modules/core/client/css/core.css

```diff
@@ -35,5 +35,10 @@ a:hover .header-profile-image {
 }
 h1 {
   font-size: 80px;
-  align="center";
-}
\ No newline at end of file
+  align: center";
+}
+
+#spacing{
+  margin-top: 50px;
+
+}
```

Comment:
can you name this something more easy to understand ?? like spaceingForXX or something like it?

Reply:
@Bakushin10 That is the specific id that @pryanalewis chose for one of the <div>s on the myclasses page. I will ask her about it.